hawkbit-repository/hawkbit-repository-jpa/src/main/java/org/eclipse/hawkbit/repository/jpa/JpaRolloutManagement.java

```diff
@@ -730,4 +752,38 @@ public void triggerNextGroup(final long rolloutId) {
         startNextRolloutGroupAction.exec(rollout, latestRunning);
     }
 
+    private TargetCount calculateTargetsAndFilterBasedOnRetriedRollout(final String targetFilter, final Long createdAt, final Long dsTypeId) {
```

Comment:
improper name = could be not based on retried

Reply:
Don't see need to have different method if it adds new type TargetCount (again not correctly reflects the class content)

Reply:
Different method was needed in order to escape duplication ...